packages/@expo/config-plugins/src/ios/BuildProperties.ts

```diff
@@ -70,14 +70,30 @@ export const withNewArchEnabledPodfileProps = createBuildPodfilePropsConfigPlugi
   'withNewArchEnabledPodfileProps'
 );
 
+/**
+ * A config-plugin to update `ios/Podfile.properties.json` from the `updates.useNativeDebug` in expo config
+ */
+export const withUpdatesNativeDebugPodfileProps = createBuildPodfilePropsConfigPlugin<ExpoConfig>(
+  [
+    {
+      propName: 'updatesNativeDebug',
+      propValueGetter: (config) => (config?.updates?.useNativeDebug ?? false).toString(),
```

Comment:
```suggestion
      propValueGetter: (config) => config?.updates?.useNativeDebug?.toString(),
```

same as android to prevent `updatesNativeDebug=false` properties when `updates.useNativeDebug` is not configured

Reply:
I used

```
propValueGetter: (config) => (config?.updates?.useNativeDebug === true ? 'true' : undefined)
```